src/app/containers/RadioSchedule/index.jsx

```diff
@@ -9,14 +13,76 @@ const logger = webLogger();
 
 const RadioScheduleContainer = ({ endpoint }) => {
   const { enabled } = useToggle('radioSchedule');
-  const { hasRadioSchedule } = useContext(ServiceContext);
+  const {
+    hasRadioSchedule,
+    service,
+    script,
+    dir,
+    timezone,
+    locale,
+  } = useContext(ServiceContext);
   const radioScheduleEnabled = enabled && hasRadioSchedule;
 
   const [schedule, setRadioSchedule] = useState([]);
 
+  const getProgramState = (currentTime, startTime, endTime) => {
+    // live - currentTime < endTime && currentTime > startTime
```

Comment:
The code below already says the same that this comment does. Is there a good way we could represent these comments as code?

```js
  const isCurrentlyLive = currentTime < endTime && currentTime > startTime;
  if (isCurrentlyLive) {
    return 'live';
  }

  const isOnDemandLive = currentTime > endTime;
  if (isOnDemandLive) {
    return 'onDemand';
  }

  return 'next';
```

Reply:
Done now 👍